core/templates/services/user.service.spec.ts

```diff
@@ -54,14 +52,12 @@ describe('User Service', function() {
     UrlService = $injector.get(
       'UrlService');
     // The injector is required because this service is directly used in this
-    // spec, therefore even though UserInfoObjectFactory is upgraded to
+    // spec, therefore even though UserInfoModel is upgraded to
```

Comment:
Is this comment still true?

Reply:
removed the comment as it was relating to ObjectFactory class